### PR TITLE
Add tests for new Reading properties

### LINKED ISSUE #409: Add test for persistence of new Reading data type properties
Given the new Reading properties indicating the data type of a `Reading.value`, provide or extend a blackbox test to save and retrieve these new values from the database. We need to ensure they're being stored and retrieved correctly. Persistence of these values was done in the following PR

https://github.com/edgexfoundry/edgex-go/pull/2428

### PR DESCRIPTION
**Pull request by jdharms**
Fixes #409

Adds some of the new Reading data type properties to the core-data reading
importer, and updates the get Reading test to check for the properties.

Also updates the POST to write some of these properties.

Signed-off-by: Daniel Harms <jdharms@gmail.com>

### CODE REVIEW

**Comment by jdharms** [comment]
recheck

**Review by AnthonyMBonafide — COMMENTED** [review]
LGTM! Like the other PR related to these changes, I am more comfortable with a QA engineer verifying and approving this since my setup has deviated from the CI jobs, but this looks great.

File: bin/postman-test/collections/core-data.postman_collection.json [hunk 1]
```diff
@@ -1849,7 +1849,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n\t\"origin\":1471806366619,\n\t\"device\":\"livingroomthermostat\",\n\t\"name\":\"co2\",\n\t\"value\":\"12\"\n}"
+							"raw": "{\n\t\"origin\":1471806366619,\n\t\"device\":\"livingroomthermostat\",\n\t\"name\":\"co2\",\n\t\"value\":\"12.2\",\n\t\"valueType\":\"Float32\",\n\t\"mediaType\":\"foo\",\n\t\"floatEncoding\": \"ieee23.2\"\n}"
```
Comment (jinlinGuan) [inline bin/postman-test/collections/core-data.postman_collection.json]: In this case, it seems mediaType is unnecessary for float type?

**Review by jinlinGuan — CHANGES_REQUESTED** [review]
Could you please add testcase for binary type?

File: bin/postman-test/collections/core-data.postman_collection.json (hunk 1 above)
Comment (jdharms) [reply bin/postman-test/collections/core-data.postman_collection.json]: Good catch, thanks

**Comment by jdharms** [comment]
> Could you please add testcase for binary type?

Done, thank you

File: bin/postman-test/collections/core-data.postman_collection.json [hunk 2]
```diff
@@ -1270,6 +1270,56 @@
...
+							"host": [
+								"localhost"
+							],
+							"port": "48080",
+							"path": [
+								"api",
+								"v1",
+								"event"
+							]
+						}
```
Comment (jinlinGuan) [inline bin/postman-test/collections/core-data.postman_collection.json]: Should use `{{baseUrl}}` for different environment
```
"url": {
        "raw": "{{baseUrl}}/api/v1/event",
        "host": [
                     {{baseUrl}}
        ],
        "path": [
                     "api",
                     "v1",
                     "event"
        ]
}
```

**Review by jinlinGuan — CHANGES_REQUESTED** [review]

File: bin/postman-test/collections/core-data.postman_collection.json (hunk 2 above)
Comment (jdharms) [reply bin/postman-test/collections/core-data.postman_collection.json]: Whoops, thank you.  Will fix.

File: bin/postman-test/collections/core-data.postman_collection.json [hunk 3]
```diff
@@ -1270,6 +1270,56 @@
...
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"device\": \"irCamera\",\n    \"readings\": [\n        {\n           \"device\": \"irCamera\",\n           \"name\": \"image\",\n           \"binaryValue\": \"83010203\",\n           \"mediaType\": \"image\"\n        }\n    ]\n}"
+						},
+						"url": {
+							"raw": "{{baseUrl}}/api/v1/event",
+							"protocol": "http",
```
Comment (jinlinGuan) [inline bin/postman-test/collections/core-data.postman_collection.json]: It's unnecessary  to specify `protocol`. `http` has already been included in `baseUrl`.

**Review by jinlinGuan — CHANGES_REQUESTED** [review]

File: bin/postman-test/collections/core-data.postman_collection.json (hunk 3 above)
Comment (jdharms) [reply bin/postman-test/collections/core-data.postman_collection.json]: Thanks, addressed.  I use Postman to author the tests, not sure why it added it in this case.

**Review by jinlinGuan — APPROVED** [review]
LGTM!

**Review by cloudxxx8 — APPROVED** [review]
LGTM based on Ginny's review

**Pull request merged**